test/dynamo/test_subgraphs.py

```diff
@@ -514,8 +514,8 @@ def fn(x, c1, c2, c3):
                     opt_fn(v1, a, b, c)
 
         # checking here we don't create 2^n graphs
-        self.assertEqual(cnt.frame_count, 7)
-        self.assertEqual(cnt.op_count, 10)
+        self.assertEqual(cnt.frame_count, 12)
```

Comment:
Why did the frame count increase here?

Reply:
Because we pass in a/b/c as aliases of the same value and dedup them - so we take the hit of guards and recompile when the bool value changes for one of the a/b/c